editor/src/editor/main.tsx

```diff
@@ -54,73 +68,78 @@ export function createEditor(): void {
 }
 
 export interface IEditorProps {
-	/**
-	 * The path of the project.
-	 */
-	projectPath: string | null;
-
-	/**
-	 * Defines the path to the currently edited scene path.
-	 */
-	editedScenePath?: string | null;
+  /**
+   * The path of the project.
+   */
+  projectPath: string | null;
+
+  /**
+   * Defines the path to the currently edited scene path.
+   */
+  editedScenePath?: string | null;
 }
 
 export interface IEditorState {
-	/**
-	 * The path of the project.
-	 */
-	projectPath: string | null;
-	/**
-	 * The path of the last opened scene.
-	 */
-	lastOpenedScenePath: string | null;
-	/**
-	 * Defines the list of all plugins to load.
-	 */
-	plugins: string[];
-	/**
-	 * Defines the current package manager being used by the editor.
-	 */
-	packageManager?: EditorProjectPackageManager;
-
-	/**
-	 * Defines wether or not compressed textures are enabled.
-	 */
-	compressedTexturesEnabled: boolean;
-	/**
-	 * Defines wether or not compressed textures are enabled in the preview.
-	 */
-	compressedTexturesEnabledInPreview: boolean;
-
-	/**
-	 * Defines wether or not experimental features are enabled.
-	 */
-	enableExperimentalFeatures: boolean;
-
-	/**
-	 * Defines if the project is being edited.
-	 */
-	editProject: boolean;
-	/**
-	 * Defines if the preferences are being edited.
-	 */
-	editPreferences: boolean;
+  /**
+   * The path of the project.
+   */
+  projectPath: string | null;
+  /**
+   * The path of the last opened scene.
+   */
+  lastOpenedScenePath: string | null;
+  /**
+   * Defines the list of all plugins to load.
+   */
+  plugins: string[];
+  /**
+   * Defines the current package manager being used by the editor.
+   */
+  packageManager?: EditorProjectPackageManager;
+
+  /**
+   * Defines the current project template being used by the editor.
+   */
+  projectTemplate?: EditorProjectTemplate;
```

Comment:
Looks like it's not needed anymore, do you confirm?

Reply:
i have removed projectTemplate from all places except project creation popup